src/server/services/procedures/chart/chart.js

```diff
@@ -207,22 +206,51 @@ chart.draw = function(lines, options={}){
         };
     }
 
+    if (options.logscale) {
+        const val = options.logscale;
+        if (typeof val === 'string') opts.logscale = { axes: val, base: 10 };
+        else if (Array.isArray(val)) {
+            const axes = val[0];
+            let base = val[1];
+            if (typeof axes !== 'string') throw Error('logscale axes was not a string (text)');
+            if (base !== undefined) {
+                const temp = +base;
+                if (Array.isArray(base) || !temp || temp <= 0) throw Error('Invalid logscale option: base must be a positive number');
```

Comment:
Why not use `BoundedNumber<1>` from [here](https://github.com/NetsBlox/NetsBlox/blob/master/src/server/services/input-types.js#L30) rather than reinventing the wheel?

It would look something like:
```javascript
try {
    base = InputTypes.parse.BoundedNumber(base, [1]);
} catch (err) {
    throw new Error(`Invalid logscale option: ${err.message}`);
}
```

Reply:
Sure, that'll behave more consistently too. I actually didn't think to use those outside of the jsdoc spec.